packages/core/src/modules/ledger/services/IndyLedgerService.ts

```diff
@@ -451,21 +458,43 @@ export class IndyLedgerService {
     }
   }
 
-  private async appendTaa(pool: IndyPool, request: Indy.LedgerRequest) {
+  private async appendTaa(pool: IndyPool, request: Indy.LedgerRequest, taa?: TransactionAuthorAgreement) {
```

Comment:
We pass the pool config to `IndyPool` in the constructor. If we make the taa config available on the `InydPool` we don't have to pass this value around everywhere